assets/js/modules/analytics-4/components/audience-segmentation/dashboard/AudienceSelectionPanel/index.test.js

```diff
@@ -275,6 +281,103 @@ describe( 'AudienceSelectionPanel', () => {
 		} );
 	} );
 
+	describe( 'AddGroupNotice', () => {
+		it( 'should display notice when there is a saved selection of one group', async () => {
+			registry
+				.dispatch( MODULES_ANALYTICS_4 )
+				.setConfiguredAudiences( [ 'properties/12345/audiences/3' ] );
+
+			const { getByText, waitForRegistry } = render(
+				<AudienceSelectionPanel />,
+				{
+					registry,
+				}
+			);
+
+			await waitForRegistry();
+
+			expect(
+				getByText(
+					/By adding another group to your dashboard, you will be able to compare them and understand which content brings back users from each group/i
+				)
+			).toBeInTheDocument();
+		} );
+
+		it( 'should not display notice when there is a saved selection of less than or more than one group', async () => {
```

Comment:
This describes the case where there is less than one group in the saved selection, but doesn't test it.

Please either add test coverage for the zero-selection case (maybe splitting the test into two or using `it.each()`), or amend the description.

Reply:
I've updated the test coverage using `it.each()` as recommended, thanks Tom!